portfolio/src/main/webapp/index.html

```diff
@@ -293,67 +304,78 @@ <h3>Here is a list of the most notable courses I have taken at Howard University
         <br>
-        <a href='#guest-book'><button>You have finished a path</button></a>
-    </div>
-    <div class='third-layer'>
-        <div class='gallery'>
-            <img src='Images/dataskeptic-pod.jpg'>
+        <div class="gallery">
+            <img src="Images/dataskeptic-pod.jpg" alt="Data Skeptic, a podcast about Data Science">
         </div>
-        <div class='gallery'>
-            <img src='Images/planetmoney-pod.jpg'>
+        <div class="gallery">
+            <img src="Images/planetmoney-pod.jpg" alt="Planet Money, a podcast about economics">
         </div>
-        <div class='gallery'>
-            <img src='Images/ted-pod.png'>
-            </div>
+        <div class="gallery">
+            <img src="Images/ted-pod.png" alt="Ted Radio Hour, a podcast about Ted Talks by NPR">
+        </div>
+        <br>
+        <a href="#guest-book"><button>You have finished a path</button></a>
     </div>
-    <div class='buffer'></div>  
-    <div class='third-layer' id="other-opportunities" >
-        <a id='other-opportunities'><h2>
-            My Additional Experiences</h2></a>
+
+    <div class="third-layer">
+        <a id="other-opportunities">
+            <h2>My Additional Experiences</h2>
+        </a>
         <p>I have participated in two college level hackathons 
             (one was at Google DC), was an undergraduate assistant 
             for Howard University’s Affective Biometrics lab, and volunteered 
             for the National Society of Black Engineers as a mentor for a middle 
             school VEXIQ Robotics competition team. Now I am working as a STEP 
-            intern for Google!</p>
-        <img src='Images/clutch-iq.png'>
+            intern for Google!
+        </p>
+        <img src="Images/clutch-iq.png" alt="This is the robot the Vex IQ team built">
         <br>
-        <a href='#guest-book'><button>You have finished a path</button></a>
+        <a href="#guest-book"><button>You have finished a path</button></a>
     </div>
-    <br>
+
     <div id="guest-book">
-        <a id='guest-book'><h1>
-            Guest Book</h1></a>
-        <img src="Images/profile-drawing.jpg" style="width:50%;height=auto;">
+        <a id="guest-book">
+            <h1>Guest Book</h1>
+        </a>
+        <img src="Images/profile-drawing.jpg" alt="A very accurate drawing of me" style="width:50%;height=auto;">
         <p>That’s all folks! Feel free to leave your name and short comments, questions, 
             or recommendations about this web app’s user experience. Then hit return 
             to start all over again!
         </p>
-        <form action="/comments.php" mmengtmethod="post">
+        <button onclick=addDonation()>Donate and Support!</button>
+        <br>
+        <div id="donate-container"></div>
+        <br>
+        <a href="#home"><button>Try Again</button></a>
+        <form action="/data" method="post">
+            <! -- The user input quantity feature hasn't been made--> 
```

Comment:
Is this comment still accurate? It looks like the quantity feature should be working?

If there's still a piece missing make sure you specify exactly what needs to be done to get it working.

Reply:
I was also able to get it working on Friday I just had to slightly change my url.